lib/index.js

```diff
@@ -229,7 +229,7 @@ function prod(mat, operand) {
     if (size1[1] === size2[0]) {
         for (let i = 0; i < size1[0]; i++) {
             result[i] = [];
-            for (let j = 0; j < size2[0]; j++) {
+            for (let j = 0; j < size1[0]; j++) {
```

Comment:
This would work if we are multiplying for example: `3 X 2` * `2 X 3`. Will it work for `2 X 3` * `3 X 2`?